### PR TITLE
[iOS] Fixup set_version call for CI

### PR DESCRIPTION
**Pull request by mcdurdin**

### CODE REVIEW

**Comment by keyman-server** [comment]
[OSK renders for Windows](https://build.palaso.org/repository/download/KeymanDesktop_TestPrRenderOnScreenKeyboards/161286:id/kvk/index.html)

**Comment by mcdurdin** [comment]
(Failing build is okay; I cancelled it because I am impatient!)

File: resources/shellHelperFunctions.sh [hunk 1]
```diff
@@ -225,3 +225,17 @@ write_download_info() {
   echo "  \"size\": \"${FILE_SIZE}\"" >> "$DOWNLOAD_INFO_FILEPATH"
   echo } >> "$DOWNLOAD_INFO_FILEPATH"
 }
+
+# set_version sets the file version on mac/ios projects
```
Comment (darcywong00) [inline resources/shellHelperFunctions.sh]: Are you changing the mac build.sh script to use `set_version`?

Comment (mcdurdin) [reply resources/shellHelperFunctions.sh]: Not for now. Only the two iOS build scripts -- Keyman and FirstVoices Keyboards

**Pull request merged**